CRM/Member/Form/Membership.php

```diff
@@ -224,7 +224,7 @@ public function preProcess() {
     }
 
     $this->assign('customDataType', 'Membership');
-    $this->assign('customDataSubType', $this->_memType);
+    $this->assign('customDataSubType', $this->getMembershipValue('membership_type_id'));
```

Comment:
@eileenmcnaughton would this work when creatng a membership?

Reply:
@seamuslee001 I tested in create mode & it correctly returns NULL from this function

Reply:
This fix works for me on a dev copy running Civi 5.75.0 on Drupal 10.3.1.

Reply:
ok guess this is fine @eileenmcnaughton I was just noticing a few lines up that memType was getting set in some cases still but i'm guessing the JS will handle that so ok to merge